tests/system_tests_bad_configuration.py

```diff
@@ -112,11 +113,11 @@ def wait_for_unresolvable_host(self):
         with open('../setUpClass/test-router.log', 'r') as router_log:
             log_lines = router_log.read().split("\n")
             expected_errors = [
-                "proton:io Name or service not known",  # Linux
-                "proton:io unknown node or service",  # macOS
+                "Connection to %s" % self.unresolvable_host_name
             ]
             errors_caught = [line for line in log_lines
-                             if any([expected_error in line for expected_error in expected_errors])]
+                             if any([expected_error in line for expected_error in expected_errors])
+                             and "failed" in line]
 
             self.error_caught = any(errors_caught)
```

Comment:
Just a thought. The test does not check that Dispatch tries to connect repeatedly; it is enough to see the log line once.

Reply:
The connection retry is on a five second cadence today. The test polls more often than that and should see the first failed connection. The code accepts one or more logs indicating failure.

Reply:
Yeah, thats what I meant. If dispatch (due to a bug) stopped retrying the connection, the test would still pass.